No movie actors in Movie -> Actors

tombriden opened the pull request:
well, what the title says. There are no actors listed/found in Movies -> Actors, while they are shown nicely in the movie details.

tthomas48 commented:
This one is unfortunately a known issue. This didn't get into the API for Frodo.  Here's the improvement request:

http://trac.xbmc.org/ticket/13658

freezy commented:
Suppose we'll remove the actors tab then.

tombriden committed:
Call updateProgress of now playing screen even when paused so the seek bar times still display
New boolean parameter in updateProgress to determine whether to display play or paused icon

tombriden committed:
commented out actor tabs from movies/tv while the api functions to get them don't exist
